ansible/host_vars/localhost/vars.yml

```diff
@@ -0,0 +1,5 @@
+---
+gh_token: '{{ vault_gh_token }}'
+namecheap_user: '{{ vault_namecheap_user }}'
+namecheap_token: '{{ vault_namecheap_token }}'
+namecheap_jumphosts: '{{ vault_namecheap_jumphosts }}'
```

Comment:
Note to self. Adjust my roles using the `vault_` convention when referencing vaulted variable (maybe also add it to readme).